### ISSUE TITLE
Documentation - PDF Error - Missing tooltip in form field

### ISSUE THREAD

**Issue by 1Copenut** [department-of-veterans-affairs/va.gov-team#15546]
This will be the spike ticket for writing documentation for missing tooltips ( https://dequeuniversity.com/rules/comply-pdf/1.0/form-no-tooltip )
